crates/router/src/configs/settings.rs

```diff
@@ -83,6 +83,7 @@ pub struct Settings {
     pub dummy_connector: DummyConnector,
     #[cfg(feature = "email")]
     pub email: EmailSettings,
+    pub supported_payment_methods_for_mandate: SupportedPaymentMethodsForMandate,
```

Comment:
In settings, new things where added. Add the below this too.
- Add info in config.example.toml.
- Basic configs in development.toml